src/main/java/org/fsin/matomat/rest/model/Order.java

```diff
@@ -1,22 +1,30 @@
 package org.fsin.matomat.rest.model;
 
 public class Order extends Transaction {
-    private Product[] products;
-    private long buy_cost;
+    private int admin;
+    private OrderedProduct purchased[];
```

Comment:
This looks like C Syntax. Array Braces should be on the type. Does this compile?

Reply:
> You can also place the brackets after the array's name:
> // this form is discouraged
> float anArrayOfFloats[];
> However, convention discourages this form; the brackets identify the array type and should appear with the type designation.

from [Java Tutorials](https://docs.oracle.com/javase/tutorial/java/nutsandbolts/arrays.html)